Resources/Views/Authentication/create_events.leaf

```diff
@@ -0,0 +1,48 @@
+#extend("base"):
+    #export("title", "SwiftLeeds")
+    #export("content"):
+        #extend("Shared/_header")
+        <div class="row">
+            <form method="POST" action="/api/v1/events/create" enctype="application/x-www-form-urlencoded">
+                <h1>Create presentation</h1>
+                <br>
+                <div class="mb-3">
+                  <label for="title" class="form-label">Event Name</label>
+                  <input class="form-control" name="name" id="name">
+                </div>
+                <div class="mb-3">
+                  <label for="date" class="form-label">Event Date</label>
+                  <input id="datepicker" class="form-control"/>
+                  <input id="date" name="date" type="hidden"/>
+                </div>
+                <div class="mb-3">
+                  <label for="location" class="form-label">Event Location</label>
+                  <input class="form-control" name="location" id="location">
+                </div>
+                <button type="submit" class="btn btn-primary">Submit</button>
+            </form>
+            <div class="col-6">
+              <img src="/img/header-bg.png" class="child-img">
+            </div>
+        </div>
+        <script src="https://cdn.jsdelivr.net/npm/@easepick/bundle@1.2.0/dist/index.umd.min.js"></script>
```

Comment:
#import("scripts") in the base to their so you can extend it to separate out the javascript from the html